packages/react-devtools-shared/src/devtools/views/Profiler/SidebarCommitInfo.js

```diff
@@ -102,11 +114,18 @@ export default function SidebarCommitInfo(_: Props) {
             </li>
           )}
 
+          {updaters !== null && commitTree !== null && (
+            <li className={styles.ListItem}>
+              <label className={styles.Label}>What caused this render</label>?
```

Comment:
I guess it's technically correct to use "render" here since we're looking at a "render" that was committed? Though it might be confusing to use "render" when we're looking at individual commits.

```suggestion
              <label className={styles.Label}>What caused this commit</label>?
```

Generally, I'm not so sure anymore about the terms "render", "update" and "commit" as they're used in devtools code.

1. Is "update" an internal synonym of "commit"?
1. Is "update" synonymous to "render"?
1. Multiple updates (e.g. `setState`) can result in a single render. Multiple renders can lead to a single commit?

Reply:
Yeah, good points. I think maybe "update" might be a safer word.